app/controllers/action_plans_controller.rb

```diff
@@ -3,6 +3,7 @@ class ActionPlansController < ApplicationController
 
   load_and_authorize_resource
   respond_to :html, :json
+  before_action :load_participation_process
```

Comment:
This should be `participatory_process`. Wouldn't you put this loading into a `concern`?